bin/asinit

```diff
@@ -210,7 +210,7 @@ function ensurePackageJson() {
   const entryPath = path.relative(projectDir, entryFile).replace(/\\/g, "/");
   const buildUntouched = "asc " + entryPath + " -b build/untouched.wasm -t build/untouched.wat --sourceMap --validate --debug";
   const buildOptimized = "asc " + entryPath + " -b build/optimized.wasm -t build/optimized.wat --sourceMap --validate --optimize";
-  const buildAll = "npm run asbuild:untouched && npm run asbuild:optimized";
+  const buildAll = "function build_all() { npm run asbuild:untouched -- $* && npm run asbuild:optimized -- $*; }; build_all";
```

Comment:
This looks like it'll break on Windows :(

Reply:
Wait, Windows? I thought this required a POSIX shell (i.e. Cygwin or WSL). Even the previous version used ``&&`` which I didn't think was understood by traditional ``cmd.exe``.

Reply:
Yeah, there's not much bash and cmd have in common, but `&&` works :)

Reply:
In that case, would you accept executing a new .js file in node? It seems like overkill, but I don't know of any other way to support Windows...

Reply:
I guess it could be implement via `concurrently` package and some extra script like here: https://github.com/kimmobrunfeldt/concurrently/issues/33#issuecomment-433084589

But it's extra dependency and isn't it too overcomplicated? Hmm